mt.json

```diff
@@ -0,0 +1,10 @@
+{
+  "authors": [
+    "Corbin Simpson", "Justin Noah <justinnoah@gmail.com>"
+  ],
+  "name": "mt-json",
+  "paths": ["."],
+  "entrypoint": "src/json",
+  "version": "0.1.0",
```

Comment:
I am *very* much in favor of semver. It's a lovely and logical way to manage versioning. I asked about a regex module in monte a bit ago because I want to implement a semver module.